packages/react/src/DispatchField.tsx

```diff
@@ -27,9 +27,9 @@ import { connect } from 'react-redux';
 import * as _ from 'lodash';
 import { UnknownRenderer } from './UnknownRenderer';
 import {
-    DispatchFieldProps,
-    formatErrorMessage,
-    mapStateToDispatchFieldProps,
+  DispatchFieldProps, DispatchFieldStateProps,
```

Comment:
formatting, but should be fine, as it will be fixed by prettier

Reply:
Ok, great